src/components/Divider.js

```diff
@@ -6,10 +6,34 @@ import {
   headRank
 } from '../util.js'
 
-export const Divider = ({ thoughtsRanked }) => <div className='divider-container'>
-  <div className={classNames({
-    divider: true,
-    // requires editable-hash className to be selected by the cursor navigation via editableNode
-    ['editable-' + hashContext(thoughtsRanked, headRank(thoughtsRanked))]: true,
-  })} />
-</div>
+export const Divider = ({ thoughtsRanked }) => {
+
+  const dividerSetWidth = React.createRef();
+
+  // get max width of nearby for divider list child elements, add 30 px and set this width for divider
+  const setStyle = () => {
+    if(dividerSetWidth.current) {
+      const parentUl = dividerSetWidth.current.closest("ul");
+      const children = parentUl.childNodes;
+      let maxWidth = 0;
+      children.forEach((child) => {
```

Comment:
This project requires a functional programming style, as clearly communicated in the [Contributing Guidelines](https://github.com/cybersemics/em/blob/dev/CONTRIBUTING.md). 

Please use `reduce` here instead of a mutable variable and a forEach loop.